verification/api/challengeresponsesession.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type Status uint8
+
+const (
+	StatusWaiting Status = iota
+	StatusProcessing
+	StatusComplete
+	StatusFailed
+)
+
+func (o Status) String() string {
+	switch o {
+	case StatusWaiting:
+		return "waiting"
+	case StatusProcessing:
+		return "processing"
+	case StatusComplete:
+		return "complete"
+	case StatusFailed:
+		return "failed"
+	}
+	return "unknown"
+}
+
+func (o *Status) FromString(s string) error {
+	switch s {
+	case "waiting":
+		*o = StatusWaiting
+	case "processing":
+		*o = StatusProcessing
+	case "complete":
+		*o = StatusComplete
+	case "failed":
+		*o = StatusFailed
+	default:
+		return fmt.Errorf("unknown status %s", s)
+	}
+	return nil
+}
+
+func (o Status) MarshalJSON() ([]byte, error) {
+	return json.Marshal(o.String())
+}
+
+func (o *Status) UnmarshalJSON(b []byte) error {
+	var s string
+
+	if err := json.Unmarshal(b, &s); err != nil {
+		return err
+	}
+
+	return o.FromString(s)
+}
+
+type Blob struct {
+	Type  string `json:"type"`
```

Comment:
I suppose it is media type? if yes can it be named as mtype for json  type?

Reply:
And please explain with comment that it is Media Type

Reply:
This is the `EvidenceBlob` described in the OpenAPI doc: https://github.com/veraison/docs/blob/main/api/challenge-response/schemas/components.yaml